File: dotnet/test/Azure.Iot.Operations.Protocol.MetlTests/DefaultTestCase.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

namespace Azure.Iot.Operations.Protocol.MetlTests
{
    public class DefaultTestCase
    {
        public DefaultTestCase()
        {
            Prologue = new();
            Actions = new();
        }

        public DefaultPrologue Prologue { get; set; }

        public DefaultAction Actions { get; set; }
    }
}

File: rust/azure_iot_operations_protocol/tests/metl/default_test_case.rs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

use serde::Deserialize;

use crate::metl::default_action::DefaultAction;
use crate::metl::default_prologue::DefaultPrologue;

#[derive(Deserialize, Debug)]
pub struct DefaultTestCase {
    #[serde(rename = "prologue")]
    pub prologue: Option<DefaultPrologue>,

    #[serde(rename = "actions")]
    pub actions: Option<DefaultAction>,
}
